server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -724,8 +709,8 @@ private String getRootUserName(SiteConfiguration siteConfig, Opts opts) throws I
       return DEFAULT_ROOT_USER;
     }
 
-    LineReader c = getLineReader();
-    c.getTerminal().writer().println("Running against secured HDFS");
+    Console c = System.console();
```

Comment:
```suggestion
    
```

Can drop this and instead do 

```
System.console().readLine("Principal (user) to grant administrative privileges to : "); 
```
below where c is used (it is used only once).

Reply:
Cleaned it up.